client/k8s/informer/K8sInformerFactory.go

```diff
@@ -0,0 +1,126 @@
+package informer
+
+import (
+	"github.com/devtron-labs/devtron/api/bean"
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"sync"
+	"time"
+)
+
+func NewGlobalMapClusterNamespace() map[string]map[string]string {
+	globalMapClusterNamespace := make(map[string]map[string]string)
+	return globalMapClusterNamespace
+}
+
+type K8sInformerFactoryImpl struct {
+	logger                    *zap.SugaredLogger
+	globalMapClusterNamespace map[string]map[string]string
```

Comment:
can you please add dample data/ some explanation about what's inside the map in comment

Reply:
done